qemu/tests/cfg/balloon_hotplug.cfg

```diff
@@ -13,6 +13,18 @@
     balloon_type_evict = evict
     balloon_type_enlarge = enlarge
     balloon_device = virtio-balloon-pci
+    Windows:
+        cdroms = "virtio"
+        # Please change following cdrom_virtio to the right version
+        # if necessary
+        cdrom_virtio = isos/windows/virtio-win.iso
+        # balloon service related cmd, which will be covered by different
+        # guest's paths in guest-os cfg.
+        install_balloon_service = "%s:\Balloon\GUEST_OS\amd64\blnsvr.exe -i"
```

Comment:
cmd will be covered by shared/cfg/guest-os/Windows/Win10/i386.cfg, why we still need install_balloon_service in this cfg ? Is it as an init cfg?

Reply:
balloon_service cmd is only available for balloon_service, balloon_hotplug cases as defined in https://github.com/avocado-framework/avocado-vt/pull/1422, so need to add them in specific cases